### PR TITLE
feat/YEET-SQTA-130-Edit_Pet_Information

### PR DESCRIPTION
**Pull request by Floudrai**

### CODE REVIEW

File: src/test/java/org/springframework/samples/petclinic/web/PetControllerTests.java [hunk 1]
```diff
@@ -171,4 +171,14 @@ void testListAllPetsCorrectInfo() throws Exception {
                 any(Pet.class)
             )));
     }
+    @Test
```
Comment (Yuq140) [inline src/test/java/org/springframework/samples/petclinic/web/PetControllerTests.java:189]: Can you put a screenshot of your unit test?
Also, maybe add another unit test? I think that only one test will not be enough

**Comment by Yuq140** [comment]
Did you do any regression testing with the latest code from master?
Please put a screenshot of the regressing testing in your pull request's description

**Comment by Floudrai** [comment]
testcases

**Review by Yuq140 — APPROVED** [review]
LGTM

**Review by saoudmessaoudi — APPROVED** [review]

**Pull request merged**